src/types.ts

```diff
@@ -40,8 +40,18 @@ type AgentChildProcessOutput =
       error: POJO;
     };
 
+type TableRow = Record<string, any>;
+
+interface TableOptions {
+  columns?: Array<string> | Record<string, number>;
+  includeHeaders?: boolean;
+  includeRowCount?: boolean;
+}
+
 export type {
   AgentStatusLiveData,
   AgentChildProcessInput,
   AgentChildProcessOutput,
+  TableRow,
+  TableOptions,
 };
```

Comment:
Since these are more generic types, put them at the top of the exports list.

Reply:
resolved by moving types up